umbraco/Umbraco-CMS
# Content migration - typed labels based on alias rather than ID

### Prerequisites


If there's an existing issue for this PR then this fixes #5908

### Description
During content migration, AddTypedLabels currently assumes that the properties to update exist with the standard IDs used in a fresh install. If the site has been upgraded from an old version then those IDs may be used by other properties which should not be updated. If the user has created their own media or member types with the standard property aliases, those properties will not be updated.

Testing:
- In 7.15..0, copy the default Image media type and create an item using the new type.
- Edit the original Image media type. Change the alias of the `umbracoWidth` property to `test` and switch it to a Textbox data type. Create a media item with a non-numeric value for that property.
- Upgrade to 8.1.0.
- Confirm that the width, height and size properties of the copied media type are updated to use the new typed labels
- Confirm that the `test` property of Image is unchanged.
